README.md

```diff
@@ -8,10 +8,14 @@ This repo contains the cli for managing tinlake pools along with an on chain reg
 
 ## Options
 The following environment variables should be set:
-* ETH_NETWORK: `ethlive` or `kovan`
+* NETWORK: `mainnet` or `kovan`
```

Comment:
we have both ETH_CHAIN and NETWORK atm which seems redundant -- we should consolidate this naming with our other repos

Reply:
yeah we should. I think we use ETH_CHAIN more often?